### PR TITLE
Adopt readme for the SAP internal report

### PR DESCRIPTION
**Pull request by albertmink**
Some minimal documentation for changes introduce in #92

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ Detailed information including third-party components and their licensing/copyri
 Pull the source code with [abapGit](https://github.com/abapGit/abapGit/) into your ABAP system.
 The latest abapGit build can be downloaded [here](https://raw.githubusercontent.com/abapGit/build/main/zabapgit.abap).
 
+The reports and their purpose
+* `z_generate_json_schema` generates the ABAP file formats JSON Schema (minimal requirement is v7.54)
+* `z_generate_file_format_folder` generates files and folder hierachy for upload and maintain the [abap-file-formats](https://github.com/SAP/abap-file-formats) repository (requires abap file format framework)
```
Comment (huber-nicolas) [inline README.md:23]: typo: hierarchy

**Review by huber-nicolas — COMMENTED** [review]
We also need the option to print the simple transformation anywhere. Maybe in z_generate_json_schema. But then the name is a bit misleading.
But for me it would be better to only have one report which can print everything (ST, Schema, Github folder)

File: README.md (hunk 1 above)
Comment (albertmink) [inline README.md:23]:
```suggestion
* `z_generate_file_format_folder` generates files and folder hierarchyfor upload and maintain the [abap-file-formats](https://github.com/SAP/abap-file-formats) repository (requires ABAP file format framework)
```

File: README.md [hunk 2]
```diff
@@ -18,6 +18,10 @@ Detailed information including third-party components and their licensing/copyri
 Pull the source code with [abapGit](https://github.com/abapGit/abapGit/) into your ABAP system.
 The latest abapGit build can be downloaded [here](https://raw.githubusercontent.com/abapGit/build/main/zabapgit.abap).
 
+The reports and their purpose
+* `z_generate_json_schema` generates the ABAP file formats JSON Schema (minimal requirement is v7.54)
+* `z_generate_file_format_folder` generates files and folder hierarchy for upload and maintain the [abap-file-formats](https://github.com/SAP/abap-file-formats) repository (requires ABAP file format framework)
```
Comment (albertmink) [inline README.md:23]:
```suggestion
* `z_generate_repo` generates files and folder hierarchy for upload and maintain the [abap-file-formats](https://github.com/SAP/abap-file-formats) repository (requires ABAP file format framework)
```

**Review by huber-nicolas — APPROVED** [review]

**Pull request merged**